package.json

```diff
@@ -60,7 +60,8 @@
     "nouislider": "^14.7.0",
     "rxjs": "~6.6.7",
     "tslib": "^2.3.1",
-    "zone.js": "~0.11.4"
+    "zone.js": "~0.11.4",
+    "memo-decorator": "^2.0.1"
```

Comment:
Why is this added? it's not used anywhere?

Reply:
It's a peer dependency for the ui library. This caused the showcase to fail on startup, no idea why this wasn't added earlier.